cypress/integration/datasetSharing.spec.js

```diff
@@ -22,6 +23,26 @@ describe('test dataset sharing', () => {
     cy.get('a > .MuiButtonBase-root').click();
 
     cy.wait(['@getDataset', '@getDatasetPolicies', '@getBillingProfileById']);
-    cy.get('[data-cy="manageAccessContainer"]').should('not.exist');
+    cy.get('[data-cy="roles-tab"]').should('not.exist');
+  });
+  it('A steward can add and remove a user to a dataset', () => {
+    cy.get('[placeholder="Search keyword or description"]').type('V2F_GWAS');
+    cy.contains(/V2F_GWAS_Summary_Stats|V2F_GWAS_Summary_Statistics/g).should('be.visible');
+    cy.contains(/V2F_GWAS_Summary_Stats|V2F_GWAS_Summary_Statistics/g).click({ force: true });
```

Comment:
We've had issues with `click({ force: true })` hiding errors in some cases - I'm not entirely sure we've come down on one side or the other of this question.

Reply:
Ah, yeah, it does seem a little hacky. This was copied from another spot in our E2E tests, so I'm guessing we've run into issues with successfully clicking this particular spot so I thought I would keep it. But, good to call out.